comp/api/api/utils/common.go

```diff
@@ -7,11 +7,22 @@
 package utils
 
 import (
+	"bufio"
+	"bytes"
 	"encoding/json"
+	"fmt"
+	"io"
 	"net"
 	"net/http"
+	"os"
+	"path/filepath"
+	"time"
 
+	pkgconfig "github.com/DataDog/datadog-agent/pkg/config"
+
+	"github.com/DataDog/datadog-agent/pkg/api/util"
```

Comment:
Would recommend aliasing this to `apiutil` to make usages more obvious:

```suggestion
	apiutil "github.com/DataDog/datadog-agent/pkg/api/util"
```